base/libgit2/callbacks.jl

```diff
@@ -82,7 +82,8 @@ function credentials_callback(cred::Ptr{Ptr{Void}}, url_ptr::Cstring,
         end
 
         # first try ssh-agent if credentials support its usage
-        if creds[:usesshagent, credid] == "Y"
+        if creds[:usesshagent, credid] === nothing ||
```

Comment:
right. don't need to do so right this minute unless that turns out to be the best fix to the blocker bugs we're seeing, but it's going to be error prone until we do